gammapy/background/reflected.py

```diff
@@ -5,24 +5,69 @@
 import numpy as np
 from numpy import sin, cos, arctan2, sqrt
 from ..image import exclusion_distance, lookup
+from astropy.table import Table
 
 __all__ = [
     'ReflectedRegionMaker',
-    'ReflectedBgMaker',
+    'CircularOffRegions',
+    'find_reflected_regions_for_event_list'
 ]
 
 
+class CircularOffRegions(Table):
+    """List of circular OFF regions
+
+    Parameters
+    ----------
+    data_rows : `~np.array`
+        Array containing the circle definitions: x, y, radius
+    """
+
+    @property
+    def number_of_regions(self):
+        """Number of OFF regions"""
+        return len(self)
+    
+    def info(self):
+        """Summary info string."""
+        s = super(CircularOffRegions, self).__repr__()
+        return s
+
+    def to_ds9(self, filename):
+        """Write ds9 regions file"""
+
+        fmt = 'fk5; circle({x},{y},{r})\n'
+        with open(filename, 'w') as fh:
+            for row in self:
+                x = row['x']
+                y = row['y']
+                r = row['r']
+                line = fmt.format(x=x,y=y,r=r)
+                fh.write(line)
+
+
 class ReflectedRegionMaker(object):
     """Finds reflected regions.
 
-    TODO: At the moment only works for circular regions!
+    More info on the reflected regions background estimation methond
+    can be found in [Berge2007]_
 
+    TODO: At the moment only works for circular regions!
     TODO: should work with world or pixel coordinates internally!???
+
+    Parameters
+    ----------
+    exclusion : ImageHDU
+        Excluded regions mask
+    fov : `~astropy.coordinates.SkyCoord`
```

Comment:
`fov` as a variable name to represent the FOV center is pretty confusing. It could also be the FOV radius or region.
Maybe call this `pointing_position` or `pnt_coord` if you want something shorter?